server/controllers/cohort.js

```
@@ -65,37 +68,38 @@ const getArchivedCohortsMetaData = (req, resp) => {
   const {
     user: { _id: userId }
   } = req;
+
   Cohort.find(
     {
       ownerIds: userId,
       isArchived: true
     },
     '_id name description favIds isArchived memberIds ownerIds',
-    { sort: { created_at: -1 } },
-    (err, docs) => {
-      if (err) {
-        return resp.status(400).send({
-          message:
-            'An error occurred while fetching the archived cohorts data. Please try again.'
-        });
-      }
-
+    { sort: { created_at: -1 } }
+  )
+    .then(docs => {
       if (!docs) {
         return resp
-          .status(404)
+          .status(400)
```

You should throw an error here - see above